src/viewport.js

```diff
@@ -929,9 +929,11 @@ $.Viewport.prototype = {
      * @param {Number} degrees The degrees to set the rotation to.
      * @param {Boolean} [immediately=false] Whether to animate to the new angle
      * or rotate immediately.
+     * @param {OpenSeadragon.Point} [pivot=false] (Optional) point in viewport coordinates
```

Comment:
I don't think we need the `= false`; the square brackets are enough to indicate that it's optional. We include the `= false` for `immediately`, so it's clear what the default is, but in the case of `pivot`, it's better to spell it out in the docs like you've done.

Reply:
Ok, I will remove the `= false`